UnanimusProject/res/layout/recommendation_activity.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/recommendation_activity"
+    style="@style/RootLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
```

Comment:
Use `style="@style/TextView` instead of `android:layout_width`, `android:layout_height`, and `android:textSize`.

Use `android:layout_gravity="center_vertical"` instead of 'android:layout_centerVertical="true"`.